src/en/news/blog/2023/v18.2.0-released/index.md

```diff
@@ -0,0 +1,523 @@
+
+   5. Confirm that only one MDS is online and is rank 0 for your FS
+
+      ceph status
+
+   6. Upgrade the last remaining MDS daemon by installing the new packages and restarting the daemon
+
+      systemctl restart ceph-mds.target
+
+   7. Restart all standby MDS daemons that were taken offline
+
+      systemctl start ceph-mds.target
+
+   8. Restore the original value of `max_mds` for the volume
+
+      ceph fs set <fs_name> max_mds <original_max_mds>
+
+6. Upgrade all radosgw daemons by upgrading packages and restarting daemons on all hosts
+
+   systemctl restart ceph-radosgw.target
+
+7. Complete the upgrade by disallowing pre-Reef OSDs and enabling all new Reef-only functionality
+
+   ceph osd require-osd-release reef
+
+8. If you set `noout` at the beginning, be sure to clear it with
+
+   ceph osd unset noout
+
+9. Consider transitioning your cluster to use the cephadm deployment and orchestration framework to simplify cluster management and future upgrades. For more information on converting an existing cluster to cephadm, see https://docs.ceph.com/en/reef/cephadm/adoption/.
+
+### Post-upgrade
+
+1. Verify the cluster is healthy with `ceph health`. If your cluster is running Filestore, a deprecation warning is expected. This warning can be temporarily muted using the following command
+
+   ceph health mute OSD_FILESTORE
+
+2. If you are upgrading from Mimic, or did not already do so when you upgraded to Nautilus, we recommend you enable the new [v2 network protocol \<msgr2\>](https://docs.ceph.com/en/reef/rados/configuration/msgr2/), issue the following command
+
+   ceph mon enable-msgr2
+
+   This will instruct all monitors that bind to the old default port 6789 for the legacy v1 protocol to also bind to the new 3300 v2 protocol port. To see if all monitors have been updated,
+
+   ceph mon dump
+
+   and verify that each monitor has both a `v2:` and `v1:` address listed.
+
+3. Consider enabling the [telemetry module](https://docs.ceph.com/en/reef/mgr/telemetry/) to send anonymized usage statistics and crash information to the Ceph upstream developers. To see what would be reported (without actually sending any information to anyone),
+
+   ceph telemetry preview-all
+
+   If you are comfortable with the data that is reported, you can opt-in to automatically report the high-level cluster metadata with
+
+   ceph telemetry on
+
+   The public dashboard that aggregates Ceph telemetry can be found at https://telemetry-public.ceph.com/.
+
+## <a id="upgrade-from-older-release"></a>Upgrading from pre-Pacific releases (like Octopus)
+
+You **must** first upgrade to [Pacific (16.2.z)](https://ceph.io/en/news/blog/2020/v16-2-0-pacific-released/) or [Quincy (17.2.z)](https://ceph.io/en/news/blog/2021/v17-2-0-quincy-released/) before upgrading to Reef.
+
+## <a id="contributors"></a>Thank You to Our Contributors
+
+The Reef release would not be possible without the contributions of the
+community:
+
+{insert contributors here}
```

Comment:
Need to insert contributors

Reply:
https://tracker.ceph.com/projects/ceph/wiki/Ceph_contributors_list_maintenance_guide